presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -2005,6 +2005,27 @@ public PartitionStatistics getStatisticsUpdate()
             return statisticsUpdate;
         }
 
+        public Table getAugmentedTableForInTransactionRead()
```

Comment:
Curious: Do we need to return non-augmented table for non transactional unpartitioned table read? Could this be generalized as the default `getTable`?

Reply:
Per discussion offline, augmentation should only happen when `tableAction` is not empty.